plugins/web/opentelemetry-instrumentation-user-interaction/src/instrumentation.ts

```diff
@@ -571,26 +587,26 @@ export class UserInteractionInstrumentation extends InstrumentationBase<unknown>
       );
     } else {
       this._zonePatched = false;
-      if (isWrapped(EventTarget.prototype.addEventListener)) {
-        this._unwrap(EventTarget.prototype, 'addEventListener');
-        api.diag.debug('removing previous patch from method addEventListener');
-      }
-      if (isWrapped(EventTarget.prototype.removeEventListener)) {
-        this._unwrap(EventTarget.prototype, 'removeEventListener');
-        api.diag.debug(
-          'removing previous patch from method removeEventListener'
+      EVENT_TARGETS.forEach(target => {
```

Comment:
IE doesn't have addEventListener() rather we should be using attachEvent(both of which hang off window (for IE)).

I think it would be better to check for the existence of "attachEvent" and if so wrap as before otherwise wrap addEventListener.

Reply:
Above is assuming that we want to support IE <= 8

Reply:
I don't think we need to worry about anything below ie11 tbh.